diff --git a/src/cmake/macros.cmake b/src/cmake/macros.cmake
index 14d9b25ac..e2b32ac8a 100644
--- a/src/cmake/macros.cmake
+++ b/src/cmake/macros.cmake
@@ -136,7 +136,6 @@ macro(cycles_external_libraries_append libraries)
     ${OPENEXR_LIBRARIES}
     ${OPENEXR_LIBRARIES} # For circular dependencies between libs.
     ${PUGIXML_LIBRARIES}
-    ${PYTHON_LIBRARIES}
     ${ZLIB_LIBRARIES}
     ${CMAKE_DL_LIBS}
   )
diff --git a/src/util/defines.h b/src/util/defines.h
index 9a161f283..d7562467d 100644
--- a/src/util/defines.h
+++ b/src/util/defines.h
@@ -28,7 +28,7 @@
 #  define ccl_device_noinline_cpu ccl_device_noinline
 
 /* Forced inlining. */
-#  if defined(_WIN32) && !defined(FREE_WINDOWS)
+#  if defined(_WIN32) && !defined(FREE_WINDOWS) && !defined(__GNUC__)
 #    define ccl_device_inline static __forceinline
 #    define ccl_device_forceinline static __forceinline
 #    define ccl_device_inline_method __forceinline
